test/Nancy.Tests/project.json

```diff
@@ -1,6 +1,8 @@
 {
+    "version": "1.4.1.0",

```

Comment:
Yeah this needs to be addressed separately. We cannot have one of our tests depend on a specific version. @jchannon